insights/tests/test_syslog.py

```diff
@@ -27,3 +33,5 @@ def test_syslog():
     assert msg_info.get('Wrapper')[0].get('message') == "--> Wrapper Started as Daemon"
     assert msg_info.get('Launching')[0].get('raw_message') == "May 18 15:13:36 lxc-rhel68-sat56 wrapper[11375]: Launching a JVM..."
     assert 2 == len(msg_info.get('yum'))
+    import pdb; pdb.set_trace()
```

Comment:
@psav you need to remove this line from the test so that it won't interfere with running tests.

Reply:
oh my goodness - did I really leave that in there.....crikey - I'm so sorry - clearly my brain was in amateur hour